packer/scripts/download_docker_images.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -euxo pipefail
+export DEBIAN_FRONTEND=noninteractive
+
+# cAdvisor
+REPO="google/cadvisor"
+VERSION=$(curl --silent "https://api.github.com/repos/$REPO/releases/latest" \
+  | grep '"tag_name":' \
+  | sed -E 's/.*"([^"]+)".*/\1/')
+
+docker pull gcr.io/cadvisor/cadvisor:"$VERSION"
+# rename for docker run later
+docker image tag gcr.io/cadvisor/cadvisor:"$VERSION" cadvisor:latest
```

Comment:
create a new tag `cadvisor:latest` so we don't need to request for $VERSION later again in terraform

Reply:
ah, yes i think it's better doing it this way. :)